changelog/unreleased/features/7367.md

```diff
@@ -0,0 +1 @@
+- Added Android 14 support.
```

Comment:
Maybe we should warn developers about behaviour related to dismissal of notifications. All other changes seems to be internal.
```suggestion
- Added Android 14 support. Android 14 users can dismiss any notification including the trip notification created by the Nav SDK, it will result in the stop of trip session.
```

Reply:
That's not true. :)
The navigation will continue, you'll receive voice instructions in background, you just won't see the notification. It's expected behaviour. Dismissal of a notification is not expected to stop the associated foreground service. They allowed it to remove the visual effect. But I'll describe it in the changelog.